pages/common/browser-sync.md

```diff
@@ -0,0 +1,19 @@
+# browser-sync
+
+> Starts local server that updates browser on file changes.
+
+- Start a server from the DIR directory:
+
+`browser-sync start --server {{DIR}} --files {{DIR}}`
+
+- Start a server from local directory, watching css file in DIR:
+
+`browser-sync start --server --files {{DIR/style.css}}`
```

Comment:
Might be better to give a wildcard pattern like "*.css". So that the user knows that multiple file watching is also supported. A single file is just a reduced example of that.